### DISCUSSION TITLE
[Question]: No version bump between releases?

### DISCUSSION THREAD

**Discussion by jerkstorecaller** [danny-avila/LibreChat#2229] [Q&A]
### What is your question?

Hi, about 20 mins ago I noticed Librechat stopped working. The page still loaded but nothing happened when I pressed Send after typing my question to GPT (basic OpenAI API key to gpt-4-turbo-preview). The containers were still up and healthy, I could browse my history, and there were no errors in the docker logs of librechat/api.

I restarted the whole compose stack, it made no difference.

I figured there was a breaking change and you must have fixed it already being as diligent as you are, so I pulled the latest images and reran docker compose up, and now everything works again. I don't really know if it was a Librechat bugfix or if OpenAI's servers were having issues, it's not worth investigating (although it would be nice if ANY error was logged in Librechat).

Anyway this GH issue is about the version number. I was previously running image a3f4782384f6 from 5 weeks ago, and am now running the latest 10ad9fc3532b from 14 hours ago. They both show version 0.6.10 in the page footer. I think you should bump the version number on new releases, especially if there's bugfixes.

### More Details

None

### What is the main subject of your question?

Documentation

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
v0.6.10 was a necessary release that unfortunately messed up the scheduling of when I cut releases due to a font licensing issue.

I'm cutting a release this week. Typically if you are not on the latest tag, it is not considered stable, and the best thing to do is use the latest commit. I do not like to go more than a month without a release but I'm addressing some specific issues before this next one.

There is a tracker here and I expect to be done Friday latest: https://github.com/danny-avila/LibreChat/milestone/2
